src/OpenTelemetry.Shims.OpenTracing/OpenTelemetry.Shims.OpenTracing.csproj

```diff
@@ -3,6 +3,7 @@
     <TargetFrameworks>$(TargetFrameworksForLibraries)</TargetFrameworks>
     <Description>OpenTracing shim for OpenTelemetry .NET</Description>
     <PackageTags>$(PackageTags);distributed-tracing;OpenTracing</PackageTags>
+    <MinVerTagPrefix>core-</MinVerTagPrefix>
```

Comment:
For this package, we had made a decision to only have beta releases as we previously considered it a non-core component. With this change, we would be redefining the stability readiness of this package as it would now begin to have alpha, beta, and rc releases (similar to Prometheus Exporter packages).

We even had an entry describing this decision in the [CHANGELOG](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Shims.OpenTracing/CHANGELOG.md#150-beta1):

> The pre-release identifier has also been changed from rc to beta as we believe this more accurately reflects the status of this package. We believe the rc identifier will be more appropriate as semantic conventions reach stability.

Do we want to change the above-mentioned expectation set for the users?

Reply:
🤷 I have thoughts and opinions, but let's discuss in our next SIG meeting. I do not plan on releasing a new version of this package today anyways.